mappings/net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig.mapping

```diff
@@ -1,5 +1,16 @@
 CLASS net/minecraft/class_2906 net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig
+	FIELD field_13221 seaLevel I
+		COMMENT Please note that you can see. seaLevel never really gets called and has NO getter to return 63.
+		COMMENT The naming is just here to keep the code clean.
 	FIELD field_13222 forcedBiome I
+		COMMENT Please note that you can see. forcedBiome never really gets called in the getter as it is hardcoded to return -1.
+		COMMENT The naming is just here to keep the code clean. Please Use getForcedBiome
+	FIELD field_13223 riverSize I
+		COMMENT Please note that you can see. riverSize never really gets called in the getter as it is hardcoded to return 4.
+		COMMENT The naming is just here to keep the code clean. Please Use getRiverSize.
+	FIELD field_13224 biomeSize I
+		COMMENT Please note that you can see. biomeSize never really gets called in the getter as it is hardcoded to return 4.
+		COMMENT The naming is just here to keep the code clean. Please Use getBiomeSize
```

Comment:
Best to leave these unmapped because we can't be sure what they are.

Reply:
I am dead sure what they are